src/main/java/uk/gov/hmcts/darts/audio/model/AudioFileInfo.java

```diff
@@ -23,5 +23,6 @@ public class AudioFileInfo {
     private Integer channel;
     @NotNull
     private Path path;
+    private boolean isTrimmed;
```

Comment:
I'm not sure about the need to add this here, I think the ViqHeader needs adding to all the files in the download zip.

This only works currently because all the files go through `trimAllToPeriod:237, OutboundFileProcessorImpl`.

Really, I think we should only need to trim the first/last audio session - so this is an existing issue, but if we change that then I think this will mess up the Viq files if they are missing the header bytes.

I think @cmagowan-hmcts is the only one that has tried to play something in a Viq Player.